airflow/providers/ssh/hooks/ssh.py

```diff
@@ -57,9 +57,14 @@ class SSHHook(BaseHook):
     :type keepalive_interval: int
     """
 
+    conn_name_attr = 'ssh_conn_id'
+    default_conn_name = 'ssh_default'
+    conn_type = 'ssh'
+    hook_name = 'SSH'
+
     def __init__(
         self,
-        ssh_conn_id: Optional[str] = None,
+        ssh_conn_id: Optional[str] = "ssh_default",
```

Comment:
```suggestion
        ssh_conn_id: Optional[str] = None,
```

This needs to be None as the connection is not strictly required for this hook as all the parameters can be passed  via Hook args. And there is a check on L98 which checks for it

Reply:
You are absolutelty right. And there was no test for it to catch it. I added a test covering this case.

Reply:
This is still to do.

Reply:
Hmm. I even added tests for it :). Was it lost during rebase ?

Reply:
No. Bad test (it did not check for ssh_conn_id none and it passed), I fixed both test and the default now.

Reply:
Now shoudl be really fixed.